### PR TITLE
:sparkles: create docker for loompy library

### PR DESCRIPTION
**Pull request by sickler-alex**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Creates a docker image for the loompy library.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Environment:
* Test files:

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [X] I have checked my code and corrected any misspellings
- [X] I have committed any related changes to the PR

### CODE REVIEW

File: loompy/2.0.16/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM python:3.9.1
+MAINTAINER Alex Sickler (sicklera@email.chop.edu)
```
Comment (migbro) [inline loompy/2.0.16/Dockerfile:2]: Please use `LABEL maintainer="Alex Sickler (sicklera@email.chop.edu)"`
As the `MAINTAINER` tag is deprecated

**Review by migbro — CHANGES_REQUESTED** [review]
Just change the maintainer syntax please. Other than that, it builds, and when run interactively, drops straight into a python prompt. I don't know enough about this so perhaps that is how it is supposed to be. I don't have a good sense of how well i will play in a cwl workflow...

File: loompy/2.0.16/Dockerfile (hunk 1 above)
Comment (migbro) [reply loompy/2.0.16/Dockerfile]: Oh, also, in general instructions should be upper case - so `ADD` instead of `add`

**Review by migbro — CHANGES_REQUESTED** [review]
Just make that last `ADD` command in all caps please

**Review by NHJohnson — APPROVED** [review]

**Review by dmiller15 — APPROVED** [review]

**Review by migbro — APPROVED** [review]
Thanks for the ADD

**Pull request merged**